<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Form</class>
    <widget class="QWidget" name="Form">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>280</width>
                <height>63</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                    <property name="spacing">
                        <number>0</number>
                    </property>
                    <item>
                        <widget class="QLabel" name="label">
                            <property name="text">
                                <string>num joint :</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QSpinBox" name="numJoint_num">
                            <property name="maximumSize">
                                <size>
                                    <width>30</width>
                                    <height>16777215</height>
                                </size>
                            </property>
                            <property name="alignment">
                                <set>Qt::AlignCenter</set>
                            </property>
                            <property name="buttonSymbols">
                                <enum>QAbstractSpinBox::NoButtons</enum>
                            </property>
                            <property name="suffix">
                                <string/>
                            </property>
                            <property name="minimum">
                                <number>1</number>
                            </property>
                            <property name="value">
                                <number>1</number>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QSlider" name="numJoint_slider">
                            <property name="minimum">
                                <number>1</number>
                            </property>
                            <property name="value">
                                <number>1</number>
                            </property>
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item>
                <widget class="QCheckBox" name="scalable_chbx">
                    <property name="text">
                        <string>Scalable</string>
                    </property>
                    <property name="checkable">
                        <bool>true</bool>
                    </property>
                    <property name="checked">
                        <bool>true</bool>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
